src/pages/cookbooks/[cookbookId]/recipes/[recipeId]/sections/[sectionId].tsx

```diff
@@ -1,11 +1,45 @@
 import { useRouter } from 'next/router';
-import { FunctionComponent } from 'react';
+import { useEffect } from 'react';
+import { useStore } from '../../../../../../store/store';
+import { CookbookSection } from '../../../../../../components/Cookbook/CookbookGuide/CookbookSection/CookbookSection';
+import { SectionModel } from '../../../../../../models/Section';
 
-export const Section: FunctionComponent = () => {
+const init = async (sectionId: string | string[]) => {
+  if (sectionId != null) {
+    // TODO: Should probably get the specific section from a section array, not the guides
+
+    useStore.getState().guides.forEach(guide => {
+      const section = guide.sections.filter(
```

Comment:
you can use `.find()` here instead so you get back a single element instead of an array